### PR TITLE
docs: add full change log link

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

图片

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    --       |
| 🇨🇳 Chinese |     --      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at fde260a</samp>

Improved the `ComponentChangelog` component to include a link to the full changelog page and support different languages. Fixed the `Drawer` component's width.

### <samp>🤖 Generated by Copilot at fde260a</samp>

* Import `Link` component from `dumi` to link to full changelog page ([link](https://github.com/ant-design/ant-design/pull/43986/files?diff=unified&w=0#diff-baac8f9e495ebe8f121c468c0b4576693b0f2998d1c333d48059c5ef385692c1R5))
* Add Chinese and English translations for `full` key in `locales` object ([link](https://github.com/ant-design/ant-design/pull/43986/files?diff=unified&w=0#diff-baac8f9e495ebe8f121c468c0b4576693b0f2998d1c333d48059c5ef385692c1R31), [link](https://github.com/ant-design/ant-design/pull/43986/files?diff=unified&w=0#diff-baac8f9e495ebe8f121c468c0b4576693b0f2998d1c333d48059c5ef385692c1R37))
* Add `extra` prop to `Drawer` component to render full changelog link with language suffix and change `width` prop to use relative value ([link](https://github.com/ant-design/ant-design/pull/43986/files?diff=unified&w=0#diff-baac8f9e495ebe8f121c468c0b4576693b0f2998d1c333d48059c5ef385692c1L166-R175))

### CODE REVIEW

**Review by li-jia-nan — APPROVED** [review]

File: .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import React, { useMemo } from 'react';
 import { createStyles } from 'antd-style';
 import { HistoryOutlined } from '@ant-design/icons';
+import { Link } from 'dumi';
```
Comment (MadCcc) [inline .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:5]: 用 Common 里的 Link？

Comment (afc163) [reply .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:5]: 是不是得全局替换一下，我看还有好多是从 dumi 中 import 的。

Comment (MadCcc) [reply .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx]: 之前想全局换一次的，但是有的参数不兼容。昨天把参数兼容了，可以换一下

File: .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import React, { useMemo } from 'react';
 import { createStyles } from 'antd-style';
 import { HistoryOutlined } from '@ant-design/icons';
 import { Button, Drawer, Timeline, Typography } from 'antd';
+import Link from '../common/Link';
```
Comment (MadCcc) [inline .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:6]:
```suggestion
import Link from '../Link';
```

**Pull request merged**